Remotion/Data/DomainObjects.Web.Test/Performance/Form.aspx.cs

```diff
@@ -16,13 +16,19 @@
 // 
 using System;
 using Remotion.Data.DomainObjects.Web.Test.WxeFunctions;
+using Remotion.Web.ContentSecurityPolicy;
 using Remotion.Web.ExecutionEngine;
 using Remotion.Web.UI;
 
 namespace Remotion.Data.DomainObjects.Web.Test.Performance
 {
   public partial class Form : WxePage
   {
+    protected override CspHeader GetCspHeader ()
+    {
+      return base.GetCspHeader().AddDirectiveValue(CspDirectives.ScriptSrc, "'nonce-testsite'");
+    }
+
```

Comment:
I would suggest moving this into the class that enables CSP. This way, the special nonce will work everywhere and we won't need to override the header on a per form basis. What do you think?